src/middleware/errorMiddleware.js

```diff
@@ -4,6 +4,23 @@ const errorMiddleware = () => next => (action) => {
   if (isPromise(action.payload)) {
     return next(action).catch(err => err);
   }
+  // return a well structured error on failure
+  if (action.type.includes('_FAILURE')) {
```

Comment:
Usually, i dont really dig .includes, using a regex test would work just fine, but its cool either way

Reply:
Yeah, I think using a regex should be more efficient.